multiapi-engine/src/main/resources/templates/asyncapi/templateSchema.ftlh

```diff
@@ -1,7 +1,9 @@
-package ${packageModel};
+package ${packageModel}.${schema.parentPackage};
```

Comment:
I am not sure, but I think you should put this in an if to check exists because I'm not sure if the content doesn't exist when the variable is filled in it will give us a null or an empty string

Reply:
In the version preivious to the merge, all the tests worked, and the changes in this pull is to ensure that his field is filled (to distinguis if the class name is in ''messages'' or ina `schema`).
If this field is empty, it should be an error.